tests/test_keras_integration.py

```diff
@@ -292,6 +292,36 @@ def test_push_to_hub(self):
 
         self._api.delete_repo(name=f"{REPO_NAME}", token=self._token)
 
+    def test_push_to_hub_tensorboard(self):
+        os.mkdir(f"{WORKING_REPO_DIR}/tb_log_dir")
```

Comment:
I think this fails because `{WORKING_REPO_DIR}` does not exist yet. You can use `os.makedirs` instead. https://github.com/huggingface/huggingface_hub/runs/5067079206?check_suite_focus=true#step:5:262